ldap_auth_provider.py

```diff
@@ -404,6 +400,72 @@ class _LdapConfig(object):
 
         return ldap_config
 
+    def _get_server(self, get_info=None):
```

Comment:
Not required, but could be nice to start adding type hints here:

```suggestion
    def _get_server(self, get_info: Optional[List[str]] = None) -> ldap3.ServerPool:
```

You'll need to import `Optional` and `List` from the `typing` module if they aren't imported already.